excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SpreadsheetValidator.java

```diff
@@ -1,26 +1,30 @@
 package uk.gov.hmcts.ccd.definition.store.excel.validation;
 
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+import java.util.stream.*;
 
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.InvalidImportException;
-import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionSheet;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.*;
 import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.MapperException;
-import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.SheetName;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.*;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.*;
 
 @Component
 public class SpreadsheetValidator {
 
+    public static final String LIST_PREFIX = "#LIST(";
+    public static final String TABLE_PREFIX = "#TABLE(";
```

Comment:
can we define an enum `DisplayContextParameter` with elements LIST, TABLE,.... inside DefinitionSheet itself?